docs/program/apis/cloud.md

```diff
@@ -188,9 +188,281 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 {{% /tab %}}
 {{< /tabs >}}
 
+### CreateLocation
+
+Create and name a {{< glossary_tooltip term_id="location" text="location" >}} under the organization and location you are currently authenticated to.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `name` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Name of the new location.
+- `parent_location_id` [(Optional[string])](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Optional parent location to put the location under.
+Defaults to to a root-level loation if you do not provide a location ID.
```

Comment:
```suggestion
Defaults to creating a location in your organization at root level if you do not provide a location ID.
```
Typo, otherwise opt-- think more explanation here would be good but I think it's still readable as is